### ISSUE TITLE
Settings accessible from the toolbar

### ISSUE THREAD

**Issue by madduck** [gorhill/uMatrix#827]
> No feature requests please

That's a real shame. I just came here to express my longing for a way to access the rules editor in the settings from the toolbar UI. I understand I should just send a PR, and I might, but not now. Can I keep this idea here for reference, and just in case someone else gets to it earlier than I do?

**Comment by gorhill** [comment]
You can access the dashboard by clicking the banner title at the top.

**Comment by madduck** [comment]
Indeed. There's even a small cogwheel that I just didn't see. Many thanks.

**Comment by gorhill** [comment]
I reopen because I want to import the same behavior as with uBO: to have the dashboard remember the last pane used.
